include/klee/util/Bits.h

```diff
@@ -88,9 +95,11 @@ namespace klee {
     // @pre((x&(x-1)) == 0)
     // @post((1 << retval) == x)
     inline unsigned indexOfSingleBit(uint64_t x) {
+      assert((x & (x - 1)) == 0);
       unsigned res = bits32::indexOfSingleBit((unsigned) (x | (x>>32)));
-      if (x&((uint64_t) 0xFFFFFFFF << 32))
-	  res += 32;
+      if (x & ((uint64_t)0xFFFFFFFF << 32))
```

Comment:
I understood your point, I'm just saying that 1) I did not modify the original code, and 2) this is not related with the current PR.
I you wish I can add it as an additional commit to improve code readability.